chia/full_node/coin_store.py

```diff
@@ -341,23 +345,24 @@ async def get_coin_records_by_parent_ids(
             return []
 
         coins = set()
-        parent_ids_db: Tuple[Any, ...]
-        if self.db_wrapper.db_version == 2:
-            parent_ids_db = tuple(parent_ids)
-        else:
-            parent_ids_db = tuple([pid.hex() for pid in parent_ids])
-        async with self.coin_record_db.execute(
-            f"SELECT confirmed_index, spent_index, coinbase, puzzle_hash, "
-            f'coin_parent, amount, timestamp FROM coin_record WHERE coin_parent in ({"?," * (len(parent_ids) - 1)}?) '
-            f"AND confirmed_index>=? AND confirmed_index<? "
-            f"{'' if include_spent_coins else 'AND spent_index=0'}",
-            parent_ids_db + (start_height, end_height),
-        ) as cursor:
-
-            for row in await cursor.fetchall():
-                coin = self.row_to_coin(row)
-                coins.add(CoinRecord(coin, row[0], row[1], row[2], row[6]))
-            return list(coins)
+        for ids in chunks(parent_ids, MAX_SQLITE_PARAMETERS):
+            parent_ids_db: Tuple[Any, ...]
+            if self.db_wrapper.db_version == 2:
+                parent_ids_db = tuple(ids)
+            else:
+                parent_ids_db = tuple([pid.hex() for pid in ids])
```

Comment:
As above, this is unrelated to the change.  Ignore if you want it in a separate PR.

```suggestion
                parent_ids_db = tuple(pid.hex() for pid in ids)
```